.prettierrc.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   semi: false,
-  singleQuote: true
+  singleQuote: true,
+  tabWidth: 2,
+  trailingComma: 'none',
```

Comment:
this ok with you to add?
Matched Font Awesome (not entirely, but on these props).